### PR TITLE
Feature/sample info

### PR DESCRIPTION
**Pull request by henrikstranneheim**

### CODE REVIEW

File: lib/MIP/Check/Pipeline.pm [hunk 1]
```diff
@@ -374,7 +374,7 @@ sub check_rd_dna {
     );
 
     ## Add to sample info
-    add_to_sample_info(
+    set_to_sample_info(
```
Comment (jemten) [inline lib/MIP/Check/Pipeline.pm]: Do we need the to? Feels like set_sample_info clearer is clearer. I'm sensing a regex has been at work here ;)

File: lib/MIP/QC/Sample_info.pm [hunk 2]
```diff
@@ -24,18 +24,18 @@ BEGIN {
...
-      add_recipe_metafile_to_sample_info
-      add_recipe_outfile_to_sample_info
-      add_to_sample_info
+      set_gene_panel
+      set_infile_info
+      set_most_complete_vcf
+      set_parameter_to_sample_info
+      set_processing_metafile_to_sample_info
+      set_recipe_metafile_to_sample_info
+      set_recipe_outfile_to_sample_info
```
Comment (jemten) [inline lib/MIP/QC/Sample_info.pm]: Same with the 'to' in the `set_SOMETHING_to_sample_info`. The to made more sense when it was add. Maybe 'in' or 'for' makes more sense? But this is nitpicking really.

Comment (henrikstranneheim) [reply lib/MIP/QC/Sample_info.pm]: Agreed! Replacing to 'in'. The 'set_in_sample_info' is a bit agnostic, but currently serves as a sub when nothing else fits. We can change that in a future PR.

**Comment by henrikstranneheim** [comment]
I had to rebase due to the VEP_RNA. Closing this PR and reopening another.

**Pull request closed without merging**